jpirumvaa/jpirumva
# #4VKc6q9X/13 Add Admin Page

#### What does this PR do?
The admin page should display the basic information about what admin can do on the page. These activities include add an article, delete the article, edit the article, and view messages sent.
#### Description of Task to be completed?
Have a UI with the features of add, delete, and edit options. Form to add article should be hidden by default and should be opened as soon as a user clicks the Add Article button.
#### How should this be manually tested?
To test this, download codes and run them to see if all features are available as specified.
#### Any background context you want to provide?
The text used in dummy text and will be replaced after fetching data from the database
#### What are the relevant pivotal tracker stories?
[https://trello.com/c/4VKc6q9X/13-admins-page]()
#### Screenshots (if appropriate)
![admin](https://user-images.githubusercontent.com/50476344/88841089-ba316b80-d1dd-11ea-932a-ed3f85e5a36d.png)
![article](https://user-images.githubusercontent.com/50476344/88841127-c5849700-d1dd-11ea-873c-f971ca499574.png)
